packages/jest-jasmine2/src/jasmine/Env.js

```diff
@@ -424,6 +424,21 @@ export default function(j$) {
     };
 
     this.it = function(description, fn, timeout) {
+      if (typeof description !== 'string') {
+        throw new Error(
+          `Invalid first argument, ${description}. It must be a string.`,
+        );
+      }
+      if (fn === undefined) {
```

Comment:
Same here

Reply:
Ok, cool. So to clarify: add an ```it.todo``` and ```test.todo``` to ignore cases like 

```test.todo('justOneArgument') ```

but still throw the error for ```test('justOneArgument')```?

Reply:
I think "yes", but it's too breaking. But I think we should do it for the next version.

(I'm all for adding `test/it.todo` now - though probably in a separate PR)

@cpojer thoughts?

Reply:
I'm not sure about `test.todo`, let's discuss that separately. I'm definitely on board with making test/it with one argument throw. I do not consider it a breaking change but rather a bug fix, so it's fine to go into a minor.